tools/dotcms-cli/cli/src/main/java/com/dotcms/cli/command/files/FilesPush.java

```diff
@@ -203,7 +203,7 @@ private Pair<Workspace, File> resolveWorkspaceAndPath() throws IOException {
         }
 
         File inputFile = this.getPushMixin().path().toFile();
-        if (!inputFile.isAbsolute()) {
+        if (!inputFile.isAbsolute() && inputFile.isFile()) {
```

Comment:
Can not be a directory?

Reply:
I think you're right. The thing  is I made a similar change on all other PuchCommands to avoid the error reported by QA
But in this case, it still can be a directory. I have a branch without these changes running to make sure.